src/ui/Font.cpp

```diff
@@ -29,8 +33,8 @@ Font::Font(const char* _name) {
 	} else {
 		path = name;
 	}
-	printf("Creating new font: %s\n", name.c_str());
 	fontstash = glfonsCreate(512, 512, FONS_ZERO_TOPLEFT);
+	//fontstash = glfonsCreate(512, 512, FONS_ZERO_BOTTOMLEFT);
```

Comment:
Note to self:  before merge, need to go through this file and clean the junk comments up.

Also need to take one more pass over the function documentation and make sure I captured the bare minimums.